source/server/admin/admin.h

```diff
@@ -259,18 +259,25 @@ class AdminImpl : public Admin,
    * Helper methods for the /config_dump url handler.
    */
   void addAllConfigToDump(envoy::admin::v3::ConfigDump& dump,
-                          const absl::optional<std::string>& mask) const;
+                          const absl::optional<std::string>& mask, const bool include_eds) const;
```

Comment:
Nit: `const` on value parameters in interfaces doesn't actually do anything, so should be dropped.

Reply:
Dropped.

Reply:
Still here :) And also in the below interface def.

Reply:
So sorry for that! Only dropped it in implementation but forgot to drop it in interface...All dropped this time